exporter/coralogixexporter/config.go

```diff
@@ -44,8 +46,50 @@ type Config struct {
 }
 
 func (c *Config) Validate() error {
-	if c.Endpoint == "" || c.PrivateKey == "" || c.AppName == "" || c.SubSystem == "" {
-		return fmt.Errorf("One of your configureation field (endpoint, private_key, application_name, subsystem_name) is empty, please fix the configuration file")
+	// check env variables
+	endpoint := c.GRPCClientSettings.Endpoint
+	privateKey := c.PrivateKey
+	appName := c.AppName
+	subSystem := c.SubSystem
+
+	if os.Getenv("CORALOGIX_ENDPOINT") != "" {
```

Comment:
I believe I left a comment earlier about the usage of env vars here. Note that it's possible to specify the same component multiple times, so, you wouldn't know to which component this env var relates to. For instance:

```
exporters:
  coralogix/department1:
    application_name: "APP_NAME"
    subsystem_name: "SUBSYSTEM_NAME"
  coralogix/department2:
    application_name: "APP_NAME"
    subsystem_name: "SUBSYSTEM_NAME"
```

When CORALOGIX_ENDPOINT or CORALOGIX_PRIVATE_KEY is specified, should it really be applied to all coralogix exporters in this collector?

Reply:
Thanks for a good idea but in Coralogix we support only one application and subsystem per user.
Multiple is not available for us.

Reply:
@open-telemetry/collector-contrib-approvers I guess I need a second opinion from another approver.

Reply:
The direct use of environment variables is not appropriate.  This should be configured in the same manner as every other configuration item for the component: through the configuration system.  While your application may not support multiple applications or subsystems per user, the collector can support an operator who wishes to configure multiple users to achieve a goal of having multiple applications and/or subsystems.

Reply:
@Aneurysm9 @jpkrohling Thanks, I will remove the environments variables support and I will update the readme accordingly.